targets/psa_switch/psa_switch.cpp

```diff
@@ -358,6 +352,12 @@ PsaSwitch::ingress_thread() {
       continue;
     }
 
+    Field &f_egress_spec = phv->get_field("psa_ingress_output_metadata.egress_port");
+    port_t egress_spec = f_egress_spec.get_uint();
```

Comment:
what's the point of the `egress_spec` variable here?

Reply:
I suspect that this code is partially a remnant of simple_switch code for handling unicast.

There eventually _should_ be something here for PSA where the unicast egress_port is copied from psa_ingress_output_metadata.egress_port into these three other places, before egress processing begins:

+ psa_egress_parser_input_metadata.egress_port
+ psa_egress_input_metadata.egress_port
+ psa_egress_deparser_input_metadata.egress_port

That does not have to be in this PR, necessarily.

Reply:
It's a remnant of simple_switch. Seems either way we can probably do without the unused "temp var"; so I'll push that as well. Sorry!! Forgetting to remove this was an oversight on my part. It's a very quick fix.

The passing that Andy mentions here will be in a separate PR.